docs/remix_commands.md

```diff
@@ -28,66 +25,22 @@ In the console, you can run the commands listed below.  Once you start to type a
 
 **remix.loadurl(url)**: Load the given url in the file explorer. The url can be of type github, swarm or ipfs.
 
-**remix.setFile(path, content)**: set the content of the file located at the given path
-
-**remix.setproviderurl(url)**: Change the current provider to Web3 provider and set the url endpoint.
```

Comment:
Same for this one

Reply:
Input: remix.setproviderurl('HTTP://127.0.0.1:7545')
returns: remix.setproviderurl is not a function

Reply:
ok